### PR TITLE
better kvs serialization

### PR DESCRIPTION
**Pull request by zzlk**

### CODE REVIEW

File: hydroflow/examples/kvs_bench/server.rs [hunk 1]
```diff
@@ -214,7 +230,7 @@ pub fn run_server(
                     -> for_each(|x| { transducer_to_peers_tx.send(x).unwrap(); });
 
                 // join for lookups
-                lookup = lattice_join::<'static, 'tick, MyLastWriteWinsRepr, MySetUnion>();
+                lookup = lattice_join::<'static, 'tick, DomPair<Max<u128>, Bottom<Fake<AutoReturnBuffer>>>, MySetUnion>();
```
Comment (zzlk) [inline hydroflow/examples/kvs_bench/server.rs:233]: that is the same thing now.

File: hydroflow/Cargo.toml [hunk 2]
```diff
@@ -8,7 +8,7 @@ documentation = "https://docs.rs/hydroflow/"
 description = "Hydro's low-level dataflow runtime and IR"
 
 [features]
-default = [ "async", "macros" ]
+default = [ "async", "macros", "kvs_bench" ]
```
Comment (zzlk) [inline hydroflow/Cargo.toml:11]: how did that get in there.

**Review by MingweiSamuel — APPROVED** [review]
The serde stuff looks annoying boilerplatey but not something to worry about

**Comment by zzlk** [comment]
> The serde stuff looks annoying boilerplatey but not something to worry about

yeah the second you need deserializeSeed everything goes out the window

**Pull request merged**